object/user.go

```diff
@@ -116,6 +116,65 @@ func GetMaskedUser(user *User) *User {
 	return user
 }
 
+func GetPrivateUser(user *User) *User {
+	org := GetOrganizationByUser(user)
+	for _, accountItem := range org.AccountItems {
+		if accountItem.Public {
+			continue
+		}
+		switch accountItem.Name {
+		case "Organization":
+			user.Owner = ""
+			break
+		case "ID":
+			user.Id = ""
+			break
+		case "Name":
+			user.Name = ""
+			break
+		case "Display name":
+			user.DisplayName = ""
+			break
+		case "Avatar":
+			user.Avatar = ""
+			break
+		case "User type":
+			user.Type = ""
+			break
+		case "Password":
+			user.Password = ""
+			break
+		case "Email":
+			user.Email = ""
+			break
+		case "Phone":
+			user.Phone = ""
+			break
+		case "Country/Region":
+			user.Region = ""
+			break
+		case "Affiliation":
+			user.Affiliation = ""
+			break
+		case "Tag":
+			user.Tag = ""
+			break
+		case "Third-party logins":
+			user.Github = ""
+			user.Google = ""
+			user.QQ = ""
+			user.WeChat = ""
+			user.Facebook = ""
+			user.DingTalk = ""
+			user.Weibo = ""
+			user.Gitee = ""
+			user.LinkedIn = ""
+			break
+		}
+	}
+	return user
```

Comment:
Too wordy, try to solve it by reflection, like [object.GetUserField()](https://github.com/casbin/casdoor/blob/master/object/user_util.go#L92).

Reply:
We can use reflection to get `Name` or `Value`, but we cannot use it set value by name like `user["Id"] = ""`

Reply:
@WindSpiritSR why can't? See:

https://github.com/casbin/casdoor/blob/c632c3c30720479e37f5fff7e9efd49b5ac6dff0/object/user_util.go#L70-L90